frontend/package.json

```diff
@@ -90,5 +90,8 @@
   "engines": {
     "npm": ">=8.9.0",
     "node": ">=18.2.0"
+  },
+  "jshintConfig": {
```

Comment:
## Add `jshintConfig`
I needed to do `8` to resolve several errors PyCharm was showing me, including `async` and `require`.

I originally was adding a new file `.jshintrc` but this seemed better, adding to `package.json`.

### How to utilize this
<img width="971" alt="Screenshot 2024-06-02 at 4 16 28 PM" src="https://github.com/jhu-bids/TermHub/assets/13045020/3440ddd9-6fdf-449b-a039-cc6a22b4674e">

- [ ] @Sigfried to enable this if useful